src/theme/index.tsx

```diff
@@ -186,6 +186,8 @@ export const TYPE = {
 }
 
 export const FixedGlobalStyle = createGlobalStyle`
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
```

Comment:
This should not be needed. We already import Montserrat through the `fontsource-montserrat` dependency